### DISCUSSION TITLE
Hyprland on Nixos with Home manager says the user systemd session is degraded

### DISCUSSION THREAD

**Discussion by vinicius507** [hyprwm/Hyprland#2167] [Q&A]
Greetings,

Everytime I rebuild my home-manager configuration with Hyprland, it prints the following message:

```
Activating reloadSystemd
The user systemd session is degraded:
  UNIT                         LOAD   ACTIVE SUB    DESCRIPTION                  
● xdg-desktop-portal-gnome.se… loaded failed failed Portal service (GNOME implem…
● xdg-desktop-portal.service   loaded failed failed Portal service
```

Does anyone know what this is? I can't seem to find anything related.

**Comment by Aqa-Ib** [comment]
I don't know about nixos but on any other distro, "xdg-desktop-portal-gnome" is incompatible with hyprland. Uninstall "xdg-desktop-portal-gnome" and use "xdg-desktop-portal-hyprland" or "xdg-desktop-portal-wlr" instead and try again. Good luck.
